src/components/AccountDrawer/AuthenticatedHeader.tsx

```diff
@@ -227,7 +228,7 @@ export default function AuthenticatedHeader({ account, openSettings }: { account
   const closeFiatOnrampUnavailableTooltip = useCallback(() => setShow(false), [setShow])
 
   const { data: portfolioBalances } = usePortfolioBalancesQuery({
-    variables: { ownerAddress: account ?? '' },
+    variables: { ownerAddress: account ?? '', chains: GQL_MAINNET_CHAINS.slice() },
```

Comment:
this will create a new array instance on every render - is that what you want? why not just pass in `GQL_MAINNET_CHAINS` as is?

Reply:
so looks like our codegen doesn't mark these as `readonly` which is more correct - i made a separate PR that helps out with that: https://github.com/Uniswap/interface/pull/6889

Reply:
GQL_MAINNET_CHAINS is readonly, the query needs to have a mutable copy of the array passed in

Reply:
the better solution here is to mark the variables accept readonly as opposed to making a copy on every render

Reply:
`usePortfolioBalancesQuery` is autogenerated by Apollo

Reply:
What we could do is in utils have a readonly array used for typing and make a copy used as a mutable array ie:

```
const GQL_MAINNET_CHAINS_READONLY = [
  Chain.Ethereum,
  Chain.Polygon,
  Chain.Celo,
  Chain.Optimism,
  Chain.Arbitrum,
  Chain.Bnb,
] as const

export const GQL_MAINNET_CHAINS_MUTABLE = GQL_MAINNET_CHAINS_READONLY.slice()

Reply:
@tinaszheng  Thank you for making the apollo change. I've resolved the merge conflicts and merged that change into my branch